packages/database/src/sequence-processor/repository.ts

```diff
@@ -24,7 +24,7 @@ export class SequenceProcessorRepository {
       .where('id', '=', id)
       .executeTakeFirst()
 
-    return row ? toRecord(row) : null
```

Comment:
Is this an intended change? `null` in this scenario is used in all repositories iirc